crt_portal/cts_forms/model_variables.py

```diff
@@ -803,3 +803,10 @@
     'something_else': [],
 
 }
+
+NOTIFICATION_PREFERENCE_CHOICES = {
+    'assigned_to': [
+        ('individual', 'Individual'),
```

Comment:
I think a tooltip would help here, something like 'Selecting "individual" means you will receive a notification email every time the action or event occurs.'

Reply:
Good point, added!